tests/test_vision_explanation.py

```diff
@@ -100,7 +118,8 @@ def test_vision_explain_loadmodel():
     _ = download_assets(modelpath)
 
     # run the main function for saliency map generation
-    res = dr.get_drise_saliency_map(imgpath, None, modelpath, 5, savepath)
+    model = (torch.load(modelpath, map_location='cpu'))
+    res = dr.get_drise_saliency_map(imgpath, model, 5, savepath)
```

Comment:
These 4 args look like the `modellocation` is 5 instead of `numclasses` (ref - lines 147-151), which may be incorrect. Suggestion having attribute names for each of the args.